packages/hardhat-core/src/internal/sentry/anonymizer.ts

```diff
@@ -226,4 +234,80 @@ export class Anonymizer {
 
     return result;
   }
+
+  private _anonymizeMnemonic(
+    errorMessage: string,
+    mnemonicPhraseLengthThreshold: number,
+    minimumAmountOfWordsToAnonymize: number
+  ): string {
+    const fragments = errorMessage.split(/[!$%^&*()_+|~\-=`{}[\]:;<>?"',.]/gu);
+    const fragmentsToBeAnonymized: string[] = [];
+    // If there are 10 consecutive words, there's a good chance of there being a mnemonic phrase
+    const wordlist = ([] as string[]).concat(
+      ...[
+        require("ethereum-cryptography/bip39/wordlists/czech"),
+        require("ethereum-cryptography/bip39/wordlists/english"),
+        require("ethereum-cryptography/bip39/wordlists/french"),
+        require("ethereum-cryptography/bip39/wordlists/italian"),
+        require("ethereum-cryptography/bip39/wordlists/japanese"),
+        require("ethereum-cryptography/bip39/wordlists/korean"),
+        require("ethereum-cryptography/bip39/wordlists/simplified-chinese"),
+        require("ethereum-cryptography/bip39/wordlists/spanish"),
+        require("ethereum-cryptography/bip39/wordlists/traditional-chinese"),
+      ].map((wordlistModule) => wordlistModule.wordlist)
+    );
+    for (const [fragmentIndex, fragment] of fragments.entries()) {
```

Comment:
Why entries? `fragmentIndex` is not used.

Reply:
Obsolete leftover :P